contrib/snort/template.py

```diff
@@ -0,0 +1,38 @@
+pkgname = "snort"
+pkgver = "3.3.5.0"
+pkgrel = 0
+build_style = "cmake"
+configure_args = [
+    "-DCMAKE_BUILD_TYPE:STRING=RelWithDebInfo",
+    "-DDNET_INCLUDE_DIR=/usr/include/dnet",
+    "-DENABLE_HARDENED_BUILD:BOOL=true",
+    "-DENABLE_SHELL:BOOL=true",
+    "-DENABLE_LARGE_PCAP:BOOL=true",
+    "-DSTATIC_LUAJIT:BOOL=true",
```

Comment:
```suggestion
    "-DENABLE_SHELL=ON",
    "-DENABLE_LARGE_PCAP=ON",
```

the hardened build passes like 4 cflags that are already the default so that doesn't do anything